src/main/scala/higherkindness/rules_scala/common/sbt-testing/SubprocessRunner.scala

```diff
@@ -10,14 +10,14 @@ object SubprocessTestRunner {
   def main(args: Array[String]): Unit = {
     val input = new ObjectInputStream(System.in)
     val request = input.readObject().asInstanceOf[TestRequest]
-
+    val arguments = Seq.empty[String] //TODO
```

Comment:
Probably need to do something here but I haven't gotten around to setting up an example of using the subprocessrunner and having something to test against. I'll be sure to do that once the current direction is given a :+1: (or, happy to do any suggestion here)